daprdocs/content/en/reference/components-reference/supported-state-stores/setup-jetstream-kv.md

```diff
@@ -0,0 +1,58 @@
+---
+type: docs
+title: "JetStream KV"
+linkTitle: "JetStream KV"
+description: Detailed information on the JetStream KV state store component
+aliases:
+  - "/operations/components/setup-state-store/supported-state-stores/setup-nats-jetstream-kv/"
+---
+
+## Component format
+
+To setup a JetStream KV state store create a component of type `state.jetstream`. See [this guide]({{< ref "howto-get-save-state.md#step-1-setup-a-state-store" >}}) on how to create and apply a state store configuration.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: state.aws.dynamodb
+  version: v1
+  metadata:
+  - name: natsURL
+    value: "nats://localhost:4222"
+  - name: jwt
+    value: "eyJhbGciOiJ...6yJV_adQssw5c" # Optional. Used for decentralized JWT authentication
+  - name: seedKey
+    value: "SUACS34K232O...5Z3POU7BNIL4Y" # Optional. Used for decentralized JWT authentication
+  - name: bucket
+    value: "<bucketName>"
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadatafield
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
+| natsURL            |        Y | NATS server address URL | "`nats://localhost:4222`"|
+| jwt                |        N | NATS decentralized authentication JWT | "`eyJhbGciOiJ...6yJV_adQssw5c`"|
+| seedKey            |        N | NATS decentralized authentication seed key | "`SUACS34K232O...5Z3POU7BNIL4Y`"|
+| bucket             |        Y | JetStream KV bucket name | `"<bucketName>"`|
+
+## Creating a JetStream bucket
+
+```bash
+nats kv add <bucketName>
```

Comment:
It is always good to say at least a sentence or two about where this command comes from, even if this is obviuos to a NATS Jetstream users. In other words say some thing about installing the NATS CLI or whatever else is needed, just to help the reader

Reply:
I added a short sentence and copied the description on how to create a nats server from pub sub component.